tools/audio/extraction/audio_tables.py

```diff
@@ -0,0 +1,108 @@
+# audio_tables.py
+# SPDX-FileCopyrightText: © 2024 ZeldaRET
+# SPDX-License-Identifier: CC0-1.0
+#
+#   Implements code tables structure and related enums
+#
+
+import struct
+from enum import IntEnum
+
+from util import incbin
+
+class AudioStorageMedium(IntEnum):
+    MEDIUM_RAM = 0
+    MEDIUM_UNK = 1
+    MEDIUM_CART = 2
+    MEDIUM_DISK_DRIVE = 3
+
+class AudioCachePolicy(IntEnum):
+    CACHE_LOAD_PERMANENT = 0
+    CACHE_LOAD_PERSISTENT = 1
+    CACHE_LOAD_TEMPORARY = 2
+    CACHE_LOAD_EITHER = 3
+    CACHE_LOAD_EITHER_NOSYNC = 4
+
+class AudioCodeTable:
+    """
+    typedef struct {
+        /* 0x00 */ s16 numEntries;
+        /* 0x02 */ s16 unkMediumParam;
+        /* 0x04 */ u32 romAddr;
+        /* 0x08 */ char pad[0x8];
+        /* 0x10 */ AudioTableEntry entries[1/* numEntries */];
+    } AudioTable; // size = 0x10 + 0x10 * numEntries
+    """
+
+    class AudioCodeTableEntry:
```

Comment:
Why is this class nested? It's a bit jarring to read and I don't think it's necessary here. And AudioCodeTable. AudioCodeTableEntry seems redundant